cluster/gce/config-default.sh

```diff
@@ -95,6 +95,21 @@ if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
   TARGET_NODE_UTILIZATION="${KUBE_TARGET_NODE_UTILIZATION:-0.7}"
 fi
 
+# Optional: Create autoscaler for cluster's pods.
+# NOT WORKING YET!
```

Comment:
Rather than say "not working" maybe explain that this feature is experimental and is not ready for production use. That way we will not have to update it every time we make it "more work"